documentation/modules/configuring/con-config-mirrormaker2-sync.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='con-config-mirrormaker2-sync-{context}']
+= Synchronizing consumer group offsets
+
+[role="_abstract"]
+Configure MirrorMaker 2 to synchronize consumer group offsets from the source cluster to the target cluster.
+
+The `MirrorSourceConnector` and `MirrorCheckpointConnector` work together using internal topics to coordinate offset tracking between clusters.
+
+`offset-syncs` topic:: Populated by the `MirrorSourceConnector`, this topic stores offset mappings between source and target clusters. By default, it's created in the source cluster.
+`checkpoints` topic:: Populated by the `MirrorCheckpointConnector` in the target cluster, this topic captures the last committed offsets for each consumer group.
```

Comment:
Do we want to mention the format of the topic names:
- `mm2-offset-syncs<SEPARATOR><CLUSTER_ALIAS><SEPARATOR>internal`
- `<CLUSTER_ALIAS><SEPARATOR>checkpoints<SEPARATOR>internal`